packages/contentstack-auth/package.json

```diff
@@ -27,7 +27,7 @@
     "@oclif/config": "^1.18.3",
     "chalk": "^4.0.0",
     "debug": "^4.1.1",
-    "inquirer": "^8.2.4",
```

Comment:
Is there any reason that we removed ^ from inquirer dependency. Getting some error because of it

@netrajpatel @antonyagustine

Reply:
@shafeeqd959 It was already communicated to you that we have some issues with 8.2.5. That's why we removed it. Kindly check your chat history for more info about this.

CC @netrajpatel